File: src/model/connection.py
import json
import mysql.connector
from mysql.connector import Error

host = "localhost"
user = "root"
password = ""
database_name = "WeGreen"

if __name__ == "__main__":
    try:
        connection = mysql.connector.connect(
            host=host, user=user, password=password, database=database_name
        )
        if connection.is_connected():
            print(json.dumps({'status': 'success', 'msg': 'Connected to the database'}))
    except Error as e:
        print(json.dumps({'status': 'error', 'msg': 'Connection failed: ' + str(e)}))

File: src/model/connection.php
<?php
$host = "localhost";
$user = "root";
$pass = "";
$dbname = "WeGreen";

$conn = new mysqli($host, $user, $pass, $dbname);
if ($conn->connect_error) {
    echo json_encode(['status' => 'error', 'msg' => 'Falha na conexão: '.$conn->connect_error], JSON_UNESCAPED_UNICODE);
    exit;
}
$conn->set_charset('utf8mb4');
$conn->query("SET NAMES 'utf8mb4' COLLATE 'utf8mb4_general_ci'");
?>
